src/elements/content-sidebar/versions/VersionsItemActions.js

```diff
@@ -54,7 +59,7 @@ const VersionsItemActions = ({
     const { can_delete, can_download, can_preview, can_upload } = permissions;
     const showDelete = can_delete && !isDeleted && !isCurrent;
     const showDownload = can_download && !isDeleted && isDownloadable;
-    const showPreview = can_preview && !isDeleted;
+    const showPreview = can_preview && !isDeleted && !isSelected;
```

Comment:
Previewing an unselected version seems to work in my testing, why are we removing it?

Reply:
It was a requested change.

Reply:
Isn't this simply removing the option to preview an already selected version in the menu?

Reply:
yep, totally misread this. This is valid!